WeakAuras/GenericTrigger.lua

```diff
@@ -1068,6 +1072,19 @@ local function trueFunction()
   return true;
 end
 
+local function parseAceEventMessageType(event) 
+  local eventType
+  for i in string.gmatch(event, "[^:]+") do
```

Comment:
You get passed in a single event if I understood that code correctly.

Reply:
the event could be: `AceEvent:TestEvent`
split = ["AceEvent", "TestEvent"]
if split[0] == AceEvent -> return split[1]
otherwise return null

Reply:
its one event but need to test if it has the aceevent prefix

Reply:
gmatch is the wrong function for it, you want match then

Reply:
I want to get the event without the prefix though - its what i'm returning

Reply:
match can do pattern matching and do that for you